### ISSUE TITLE
Response Content-Disposition filename support

### ISSUE THREAD

**Issue by acdha** [cantaloupe-project/cantaloupe#301]
I have 4.1.1 deployed and the response Content-Disposition feature almost works as documented:

https://chroniclingamerica.loc.gov/iiif/2/vtu_eggplant_ver01%2Fdata%2Fsn91066782%2F00415627919%2F1919073001%2F0200.jp2/0,5,5270,7518/572,817/0/default.jpg

```
Content-Disposition: inline; filename=vtu_eggplant_ver01_data_sn91066782_00415627919_1919073001_0200.jp2.jpg
```

Trying to set `attachment` works:

https://chroniclingamerica.loc.gov/iiif/2/vtu_eggplant_ver01%2Fdata%2Fsn91066782%2F00415627919%2F1919073001%2F0200.jp2/0,5,5270,7518/572,817/0/default.jpg?response-content-disposition=attachment
```
Content-Disposition: attachment; filename=vtu_eggplant_ver01_data_sn91066782_00415627919_1919073001_0200.jp2.jpg
```

However, trying to set a filename using [the example from the documentation](https://cantaloupe-project.github.io/manual/4.1/endpoints.html#Response%20Content%20Disposition) does not set the desired filename:

https://chroniclingamerica.loc.gov/iiif/2/vtu_eggplant_ver01%2Fdata%2Fsn91066782%2F00415627919%2F1919073001%2F0200.jp2/0,5,5270,7518/572,817/0/default.jpg?response-content-disposition=attachment%3B%20filename%3Dimage.jpg

```
Content-Disposition: attachment; filename=vtu_eggplant_ver01_data_sn91066782_00415627919_1919073001_0200.jp2.jpg
```

**Comment by adolski** [comment]
Hi @acdha , thanks for pointing this out. The tests for this feature must have been overlooked. It should be fixed now (on the `release/4.1` branch).

**Comment by acdha** [comment]
Awesome, thanks for the super-fast response!
